bin/spring

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env ruby
+
+# This file loads Spring without using loading other gems in the Gemfile, in order to be fast.
+# It gets overwritten when you run the `spring binstub` command.
+
 if !defined?(Spring) && [nil, "development", "test"].include?(ENV["RAILS_ENV"])
-  gem "bundler"
   require "bundler"
 
-  # Load Spring without loading other gems in the Gemfile, for speed.
-  Bundler.locked_gems&.specs&.find { |spec| spec.name == "spring" }&.tap do |spring|
+  Bundler.locked_gems.specs.find { |spec| spec.name == "spring" }&.tap do |spring|
```

Comment:
do not fix rubocop in config and bin files

Reply:
okey, mb its was rubocop